{
  "comments": [
    {
      "key": {
        "uuid": "818d82b8_d0963297",
        "filename": "plugins/tripleo-overcloud/introspect.yml",
        "patchSetId": 3
      },
      "lineNbr": 75,
      "author": {
        "id": 1001189
      },
      "writtenOn": "2020-09-07T08:20:04Z",
      "side": 1,
      "message": "do you need to use stat? isn\u0027t it logically known by now which one is there (i mean the tasks would fail already if copying fails) ... so if, then at most you need just set_fact and calculation based on the \u0027instack_file_path.stat.exists || ~/instackenv.json\u0027 ... as that is what previous tasks you are adding says (if not exists, do copy from inventory or fail)\n\ni wonder why we even copy provided-by-user file into different file name (as provided by user) instead of always into instackenv.json, could there exist more then one file? if we could have the name unified on UC side imo",
      "revId": "41f1846f8415c11e51f950a9ea92e7e3662bd35d",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "2160b002_2c5ad407",
        "filename": "plugins/tripleo-overcloud/introspect.yml",
        "patchSetId": 3
      },
      "lineNbr": 75,
      "author": {
        "id": 1003043
      },
      "writtenOn": "2020-09-07T09:25:43Z",
      "side": 1,
      "message": "Yes, I need.\nI tried before.\nthe issue was the register make registration even if the task fails, so I need to repeat the stat.\nstat is used for path resolve to absolute, this was the original behavior.\n\nI wanted to avoid changing the original behavior too much.\nProbably your 2th observation is right, but ..",
      "parentUuid": "818d82b8_d0963297",
      "revId": "41f1846f8415c11e51f950a9ea92e7e3662bd35d",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "4c0f0ee2_ae9e7e3c",
        "filename": "plugins/tripleo-overcloud/introspect.yml",
        "patchSetId": 3
      },
      "lineNbr": 75,
      "author": {
        "id": 1001189
      },
      "writtenOn": "2020-09-10T11:17:53Z",
      "side": 1,
      "message": "Ok, so i understand the stat again part.\n\nBut still the error fail/error message is redundant then - just dead code.\n\nMaybe you could then add ignore_error(s) ?\nThat task will in all current cases be \"skipped\" since stat.exists will be true,\nonly in case file is missing (from provisioner or user) it will show Ignored error - followed up by stat \u0026 fail: msg - so that should be ok / valid use of ignore_error (vs using failed_when never would make the task appear as ok when it wasn\u0027t.)",
      "parentUuid": "2160b002_2c5ad407",
      "revId": "41f1846f8415c11e51f950a9ea92e7e3662bd35d",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "37938921_b4d191b6",
        "filename": "plugins/tripleo-overcloud/introspect.yml",
        "patchSetId": 3
      },
      "lineNbr": 99,
      "author": {
        "id": 1003043
      },
      "writtenOn": "2020-09-09T10:05:53Z",
      "side": 1,
      "message": "stat.path",
      "revId": "41f1846f8415c11e51f950a9ea92e7e3662bd35d",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}